csv/MFT.csv

```diff
@@ -16,6 +16,7 @@ Defence Evasion,^dControl\.exe$|^gmer\.exe$|IObit Unlocker|DefenderControl|file_
 Defence Evasion,CleanWipe|nssm\.exe$|processhacker,.,,Internal,Medium
 EDR Evasion,^EDRSilencer\.exe$|^FireBlock\.exe$,.,,https://github.com/netero1010/EDRSilencer,High
 Encryption,^dcrypt|^truecrypt|^VeraCrypt,.,,Internal,Low
+Enumeration Tool - Network Scanner (NS) Tool,ns\.exe,.,,Internal,High

```

Comment:
do you think we should add in a ^ns\.exe as I think ns\.exe may pull out lots of other binaries that end in ns.exe?

Reply:
ah yes, agree.